.github/workflows/beta-tests.yml

```diff
@@ -0,0 +1,64 @@
+# Testing the code base against a specific Meilisearch feature
+name: Beta tests
+
+# Will only run for PRs and pushes to *-beta
+on:
+  push:
+    branches: ['!bump-meilisearch-v*.*.*-beta', '**-beta']
+  pull_request:
+    branches: ['!bump-meilisearch-v*.*.*-beta', '**-beta']
+
+jobs:
+  integration_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Strapi is not yet compatible with node 16
+        node: ['14', '16']
```

Comment:
16 or 18? Why there is 16 in the matrix if it didn't work?

Reply:
`16` works but `18` does not!

Reply:
Yes so maybe the sentence should be:
` # Strapi is not yet compatible with node 18` no?

Reply:
Oooh haha yes! 
```suggestion
        # Strapi is not yet compatible with node 18
        node: ['14', '16']
```